.github/actions/spelling/allow/README.md

```diff
@@ -8,11 +8,11 @@ E.g., you could include a list of system APIs here, or potential
 contributors (so that if a future commit includes their name,
 it'll be accepted).
 
-### Files
+## Files
```

Comment:
I'm starting to use a markdown linter that complains about missing hierarchy levels.